posts/GSoC_2019_Fourth_Blog.md

```diff
@@ -0,0 +1,74 @@
+---
+title: GSoC 2019 Third Blog
+date: 2019-07-20 15:03 UTC-19:03
+slug: GSoC_2019_Fourth_Blog
+tags: GSoC 2019
+description: 2019 Google Summer of Code Cantera Project Blog
+type: text
+author: Chao Xu
+---
+
+# GSoC 2019 Fourth Blog
+
+After two months, the functions of ChemCheck are comparatively robust. We improved editing page and ck2yaml error reporting page this week, and we
```

Comment:
> comparatively robust

The features are robust compared to what? To use this phrase, there has to be a comparison. Can you try to find another word to describe what you mean here?